components/atom/accordion/src/index.js

```diff
@@ -0,0 +1,94 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types'
+import MoleculeCollapsible from '../../../molecule/collapsible/src'
+
+const BASE_CLASS = 'sui-AtomAccordion'
+
+class AtomAccordion extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      childrenStateList: this.getChildrenInitialStateList()
+    }
+  }
+
+  getChildrenInitialStateList = () => {
+    return this.props.children.map(() => true)
+  }
+
+  onOpen = id => {
+    const childrenNewStatePromise = new Promise(resolve => {
+      resolve(
+        this.state.childrenStateList.map((child, index) => {
+          return index !== id
+        })
+      )
+    })
+    childrenNewStatePromise.then(childrenNewStateList => {
+      this.setState({
+        childrenStateList: [...childrenNewStateList]
+      })
+      this.forceUpdate()
```

Comment:
You shouldn't need to create a promise nor to force the update. I've seen that you are using componentWillReceiveProps within de collapsible. I think this is breaking things/not working as expected (it's also deprecated). Try getDerivedStateFromProps and/or componentDidUpdate and I'm pretty sure you won't need this promise/force hack